src/marqo/tensor_search/index_meta_cache.py

```diff
@@ -179,6 +179,7 @@ def populate_cache(config: Config):
     """
     global index_info_cache
     index_management = IndexManagement(config.vespa_client)
+    index_management.vespa_client.wait_for_application_convergence()
```

Comment:
Why do we need this? If it's for local Vespa, can we wait on convergence in entrypoint?